(Home) Performance

melcidar opened the pull request:
MAIN
 * this is just so I don't lose my progress
 * I have made charts using random data
 * The data in Number of projects, Total project value, etc., is still just added text, later I will make it so it's data
 * I have to add states and things to buttons, such as Home on side bar and 2023 performance, also drop down for year selection...
 * All in all, everything is just work in progress. 
 OTHER
 * ///// 
 
current look of the page:
![image](https://github.com/becarselman/ant-colony-pharaoh-frontend/assets/119449806/16bc2da5-091d-49c6-8d97-b8a6c8122102)
![image](https://github.com/becarselman/ant-colony-pharaoh-frontend/assets/119449806/65250bcc-9358-4c44-8b6d-1f34d1f48ab0)
![image](https://github.com/becarselman/ant-colony-pharaoh-frontend/assets/119449806/4a226f6b-1394-4a20-93ad-f05b70d8fd39)

melcidar committed:
Home - performance WIP

--------------------------------------------------------------------
 MAIN
 * this is just so I don't lose my progress
 * I have made charts using random data
 * The data in Number of projects, Total project value, etc., is still just added text,
	later I will make it so it's data
 * I have to add states and things to buttons, such as Home on side bar and 2023 performance, also drop down for year selection...
 * All in all, everything is just work in progress.
--------------------------------------------------------------------
 OTHER
 * /////
--------------------------------------------------------------------

melcidar committed:
HOME - PERFORMANCE

--------------------------------------------------------------------
 MAIN
 * minor changes to code, to make it simplier and easier to read
--------------------------------------------------------------------
 OTHER
 * /////
--------------------------------------------------------------------

melcidar committed:
Home - Performance

--------------------------------------------------------------------
 MAIN
 * moved data and options to different files
 * minor changes to charts
--------------------------------------------------------------------
 OTHER
 * ////
--------------------------------------------------------------------

melcidar committed:
Performance

--------------------------------------------------------------------
 MAIN
 * edited charts
--------------------------------------------------------------------
 OTHER
 * I have error regarding the dropdown arrow for years selection, to be fixed
--------------------------------------------------------------------

melcidar committed:
Home (Performace)

--------------------------------------------------------------------
 MAIN
 * changed some svg pictures to be just text
--------------------------------------------------------------------
 OTHER
 * still issues with error when clicking on year dropdown
--------------------------------------------------------------------

melcidar committed:
Home (Performance)

--------------------------------------------------------------------
 MAIN
 * added highlighted look to selected options
--------------------------------------------------------------------
 OTHER
 * this is most probably the last change until tomorrow's demo
--------------------------------------------------------------------

melcidar committed:
Performance page

--------------------------------------------------------------------
 MAIN
 * just small change in scss file
--------------------------------------------------------------------
 OTHER
 * ////
--------------------------------------------------------------------

melcidar committed:
Deleting console logs

--------------------------------------------------------------------
 MAIN

* deleted console logs

--------------------------------------------------------------------
 OTHER

* tried to change pie chart legend, failed
--------------------------------------------------------------------

becarselman commented on src/components/home/Home.js:
```diff
@@ -0,0 +1,114 @@
+                    className={`selection3 ${
+                        selectedOption === '2023 Plan' ? 'highlight' : ''
+                    }`}
+                    onClick={() => handleChange('2023 Plan')}
```
move this function inside of onClick outside of jsx

becarselman commented on src/components/home/Home.js:
```diff
@@ -0,0 +1,114 @@
+                        ? 'highlight'
+                        : ''
+                    }`}
+                    onClick={() => handleChange('Development & Revenue Costs')}
```
move this function inside onClick outside of jsx on all places

becarselman commented on src/components/home/Home.js:
```diff
@@ -0,0 +1,114 @@
+                </div>
+            </div>
+            <div className='cards'>
+                {data.map((item) => (
```
move map outside of jsx

becarselman commented on src/components/home/Home.scss:
```diff
@@ -0,0 +1,249 @@
+.container{
+    display: flex;
+    flex-direction: row;
+    //height: 100vh;
```
remove commented code

melcidar committed:
Performance changes

--------------------------------------------------------------------
 MAIN
 * resolving comments, moves functions and mapping from jsx
--------------------------------------------------------------------
 OTHER
 * ////
--------------------------------------------------------------------

melcidar committed:
Merge branch 'develop' into FEATURE/Meliha01/yZG9MYJq
